code/core/src/csf-tools/CsfFile.ts

```diff
@@ -116,6 +116,25 @@ const sortExports = (exportByName: Record<string, any>, order: string[]) => {
   );
 };
 
+const hasMount = (play: t.Node | undefined) => {
+  if (t.isArrowFunctionExpression(play) || t.isFunctionDeclaration(play)) {
+    const params = play.params;
+    if (params.length >= 1) {
+      const [arg] = params;
+      if (t.isObjectPattern(arg)) {
+        return !!arg.properties.find((prop) => {
+          if (t.isObjectProperty(prop) && t.isIdentifier(prop.key)) {
+            return prop.key.name === 'mount';
+          }
+        });
+      }
+    }
+  }
+  return false;
+};
+
+const MODULE_MOCK_REGEX = /^[.\/#].*\.mock(\.)?[^.]*$/i;
```

Comment:
This check seems like it'll find some false positives..

Reply:
fixed